mrp_repair_full_editable/views/mrp_repair_view.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="view_repair_order_form_fees" model="ir.ui.view">
+        <field name="name">mrp.repair.form.fees</field>
+        <field name="model">mrp.repair</field>
+        <field name="inherit_id" ref="mrp_repair.view_repair_order_form" />
+        <field name="arch" type="xml">
+            <data>
+                <field name="fees_lines" position="attributes">
+                     <attribute name="attrs">{'readonly': ['|',('state','in','done'),
```

Comment:
Lo probé y el odoo no se ha quejado, pero lo puse in por que pensaba poner los dos estados pero al final solo deje uno y no puse el =, ya esta corregido.